cli/src/commands/agent/validators.ts

```
@@ -0,0 +1,44 @@
+    return numLimit;
+  }
+
+  private static isValidUri(uri: string): boolean {
+    try {
+      new URL(uri);
+      return true;
+    } catch {
+      return false;
+    }
+  }
+}
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)